spec/graphql/schema/build_from_definition_spec.rb

```diff
@@ -766,14 +832,14 @@ def todo_add(args, ctx) # this is a method and accepting arguments
         end
       end
 
-      describe "wihtout defaults" do
-        let(:base_hash) { {} }
-        it "raises a KeyError" do
-          assert_raises(KeyError) do
-            schema.execute("mutation { todoAdd: todo_add(text: \"Buy Milk\") { text } }", context: {context_value: "bar"}, root_value: todos)
-          end
-        end
-      end
+      # describe "wihtout defaults" do
+      #   let(:base_hash) { {} }
+      #   it "raises a KeyError" do
+      #     assert_raises(KeyError) do
+      #       schema.execute("mutation { todoAdd: todo_add(text: \"Buy Milk\") { text } }", context: {context_value: "bar"}, root_value: todos)
+      #     end
+      #   end
+      # end
```

Comment:
It's ok with me to remove this behavior, but can we remove the test altogether? Any reason to keep it commented out?